src/pages/reference/providers.mdx

```diff
@@ -2,57 +2,53 @@ export const description = 'Details on the cloud providers supported by Nitric'
+
+| **Resource**                           | **AWS**                                              | **Azure**                                   | **Google Cloud**                             | **Local**                                                                          |
+| -------------------------------------- | ---------------------------------------------------- | ------------------------------------------- | -------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [APIs](/apis)                          | [API Gateway](./providers/aws/apis)                  | [API Management](./providers/azure/apis)    | [API Gateway](./providers/gcp/apis)          | [Custom](https://github.com/nitrictech/cli/blob/main/pkg/cloud/gateway/gateway.go) |
+| [Key Value Stores ](/keyvalue)         | [DynamoDB](./providers/aws/keyvalue)                 | [Table Storage](./providers/azure/keyvalue) | [FireStore](./providers/gcp/keyvalue)        | BoltDB                                                                             |
+| [Messaging: Topics](/messaging#topics) | [SNS](./providers/aws/topics)                        | [Event Grid](./providers/azure/topics)      | [PubSub](./providers/gcp/topics)             | Custom                                                                             |
+| [Messaging: Queues](/messaging#queues) | [SQS](./providers/aws/queues)                        | [Storage Queues](./providers/azure/queues)  | [PubSub](./providers/gcp/queues)             | Custom                                                                             |
+| [Schedules](/schedules)                | [CloudWatch Event Bridge](./providers/aws/schedules) | [Dapr Binding](./providers/azure/schedules) | [Cloud Scheduler](./providers/gcp/schedules) | Custom                                                                             |
+| [Secrets](/secrets)                    | [Secrets Manager](./providers/aws/secrets)           | [Key Vault](./providers/azure/secrets)      | [Secret Manager](./providers/gcp/secrets)    | Custom                                                                             |
+| [Storage](/storage#buckets)            | [S3](./providers/aws/storage)                        | [Blob Storage](./providers/azure/storage)   | [Cloud Storage](./providers/gcp/storage)     | SeaweedFS                                                                          |
+| Services                               | Lambda                                               | Container Apps                              | CloudRun                                     | Docker                                                                             |
 
 Our code is completely open-source on our [GitHub](https://github.com/nitrictech/nitric), so you can know exactly how your resources are being deployed and handled at runtime.
```

Comment:
```suggestion
Our code is completely open-source on our [GitHub](https://github.com/nitrictech/nitric), so you can know exactly how your resources are being deployed and handled at runtime.

## Custom Providers

Because nitric providers are plugins you're able to create or extend them to meet your specific deployment requirements. Common reasons for custom provider development include:

- Using a language other than Go to develop a provide (nitric providers can be written in many other languages)
- Using a specific Infratructure as Code engine, e.g. using Terraform or AWS CDK instead of Pulumi
- Changing the way resources are deployed to use a preferred service or meet regulatory or policy requirements within your organization.
- Reusing existing work on infrastructure automation, such using existing Terraform Modules for deployment with Nitric
- Deployments to another cloud provider or on-premises deployments

You can read more about building custom providers [here](/reference/providers/custom)

<Note>
Improving the ease and flexibility of custom provider development is currently a priority for the Nitric team, if you need help or want to request a feature get in touch on [GitHub](https://github.com/nitrictech/nitric) or [Discord](https://nitric.io/chat).
</Note>
```

Reply:
Applied and fixed link to building custom providers